docs/getting_started/installation.rst

```diff
@@ -207,6 +207,7 @@ Once cloned, ``cd`` into the ``modin`` directory and use ``pip`` to install:
 
   cd modin
   pip install -e .
+  pip install -e .[all]  # will install dependencies for all engines
```

Comment:
```suggestion
  pip install -e .[all]  # will install dependencies for all engines
```
I would rather we stub missing things tho... should speed up docs building at readthedocs

Reply:
I did not understand a little about accelerating the building of documentation. Is this code called during build?

Reply:
`pip install` thing is called during our docs build step in the CI, so reducing runtime there would reduce this step duration. Plus it would ease developers' life - they would have to install less stuff to build docs.

But this shouldn't be a blocker anyway (though running two `pip install .` is still too much)